temba/msgs/models.py

```diff
@@ -1258,6 +1287,10 @@ def as_task_json(self):
             data.update(dict(chatbase_api_key=chatbase_api_key, chatbase_version=chatbase_version,
                              is_org_connected_to_chatbase=True))
 
+        if self.metadata:
+            metadata = json.loads(self.metadata)
+            data['quick_replies'] = metadata.get('quick_replies', None)
```

Comment:
So I know we told you guys you should be passing around params like `quick_replies` in `Msg` methods - but the Msg struct passed to channels should mirror Msg database objects. So this should be adding `metadata` to `data`.